src/main/java/core/basesyntax/model/User.java

```diff
@@ -2,7 +2,7 @@
 
 import java.util.Objects;
 
-public class User {
+public class User implements Cloneable {
```

Comment:
```suggestion
public class User {
```
This is out of scope

Reply:
thanks, removed